locale-theme/it/app.po

```diff
@@ -89,7 +89,7 @@ msgid "Information on emissions and discharges (e.g. noise, energy, radiation, w
 msgstr ""
 
 msgid "Make a request"
-msgstr ""
+msgstr "Inserisci richiesta"
```

Comment:
When we pull from transifex this is going to get overridden. I suggest changing it to this in transifex, and when we email them, note that you've done so.